src/main/java/com/auth0/json/auth/TokenHolder.java

```diff
@@ -70,18 +72,27 @@ public String getTokenType() {
      *
      * @return the number of seconds in which this token will expire, since the time it was issued.
      */
-    @JsonProperty("expires_in")
     public long getExpiresIn() {
         return expiresIn;
     }
 
+    /**
+     * Get the expiration date of this token. This value is <strong>not</strong> part of the actual token response from the
+     * API, but rather is calculated and provided for convenience.
+     *
+     * @return the the date of this token's expiration.
```

Comment:
```suggestion
     * @return the date of the Access Token's expiration.
```

Reply:
👍 I've pushed a new commit with this change.